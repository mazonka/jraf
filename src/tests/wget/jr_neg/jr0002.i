command=jr logout f82d3bb17612458c