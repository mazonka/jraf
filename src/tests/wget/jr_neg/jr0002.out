JRAF_ERR bad command [logout]